File: Domain/Fretter.Domain/Dto/Canal.cs
using System;
namespace Fretter.Domain.Dto.Dashboard
{
    public class Canal
    {
        public Canal()
        {
        }

        public int Id { get; set; }
        public string CanalNome { get; set; }
        public int EmpresaId { get; set; }
    }
}

File: Web/Fretter.Web/src/app/core/models/Fusion/Canal.ts

import { Empresa } from "../empresa.model";
import { EntityBase } from "../entityBase";

export class Canal extends EntityBase {
	
	constructor(){
		super();
		this.id = 0;
    }
    inclusao: Date;
    razaoSocial: string;
    nomeFantasia: string;
    cnpj: string;
    canalNome: string;
    segmentoId: number;
    origemImportacaoId: number;
    inscricaoEstadual: string;
    empresaId: number;
    dataCadastro: Date;

}
